server/src/com/thoughtworks/go/server/service/PipelineConfigService.java

```diff
@@ -137,6 +138,13 @@ public void createPipelineConfig(final Username currentUser, final PipelineConfi
         update(currentUser, pipelineConfig, result, createPipelineConfigCommand);
     }
 
+    public void deletePipelineConfig(final Username currentUser, final String pipelineName, final LocalizedOperationResult result) {
+        PipelineConfig pipelineConfig = getPipelineConfig(pipelineName);
+        DeletePipelineConfigCommand deletePipelineConfigCommand = new DeletePipelineConfigCommand(goConfigService, pipelineConfig, currentUser, result);
+        update(currentUser, pipelineConfig, result, deletePipelineConfigCommand);
+        if(result.isSuccessful()) result.setMessage(LocalizedMessage.string("PIPELINE_DELETE_SUCCESSFUL", pipelineName));
```

Comment:
We follow the convention of surrounding if by brackets so that additional statements made later are not confusing.